packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -506,6 +509,12 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     });
   }
 
+  /* @conditional-compile-remove(unsupported-browser) */
+  public allowUnsupportedBrowserVersion(): void {
+    this.context.setState({ ...this.context.getState(), unsupportedBrowserVersionsAllowed: true });
```

Comment:
nit: can we follow the same pattern as `iIsLocalMicrophoneEnabled` and create a `setAllowUnsupportedBrowserVersion` function in the context?